guides/source/configuring.md

```diff
@@ -629,7 +629,15 @@ Defaults to `'signed cookie'`.
 
 * `config.action_dispatch.cookies_same_site_protection` configures the default
   value of the `SameSite` attribute when setting cookies. When set to `nil`, the
-  `SameSite` attribute is not added.
+  `SameSite` attribute is not added. To allow the value of the `SameSite` attribute
+  to be configured dynamically based on the request, a proc may be specified.
+  For example:
+
+    ```ruby
+    config.action_dispatch.cookies_same_site_protection = Proc.new do |request|
```

Comment:
Let's use a lambda in the example just for the sake of code convention, but it is ok to say proc in the text.

```suggestion
    config.action_dispatch.cookies_same_site_protection = ->(request) do
```